wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConnection.java

```diff
@@ -439,6 +442,30 @@ public void clone(CloneRequest request) throws GitException, UnauthorizedExcepti
                 cloneCommand.setBranchesToClone(request.getBranchesToFetch());
             }
 
+            LineConsumer lineConsumer = lineConsumerFactory.newLineConsumer();
+            cloneCommand.setProgressMonitor(new BatchingProgressMonitor() {
+                @Override
+                protected void onUpdate(String taskName, int workCurr) {
```

Comment:
This method invokes when the total work isn't known. I've added an output inside this method too